tool_collections/vcftools/vcftools_consensus/vcftools_consensus.xml

```diff
@@ -0,0 +1,82 @@
+<tool id="vcftools_consensus" name="Consensus" version="0.1.11">
+    <description>Apply VCF variants to a fasta file to create consensus sequence</description>
+
+    <requirements>
+        <requirement type="package">samtools</requirement>
+        <requirement type="package">htslib</requirement>
```

Comment:
Strictly it is not required, but it is good for reproducibility as Marius mentioned.